src/helpers/fetchGrantAddresses.ts

```diff
@@ -25,6 +25,8 @@ export async function fetchGrantsByAddress(
     .filter((a) => a != hre.ethers.constants.AddressZero)
     .filter((a) => a != "0x0000000000000000000000000000000000000001");
 
+  grantAddresses.push("0x4edb5dd988b78b40e1b38592a4761f694e05ef05");
```

Comment:
What are we using this address for?

Reply:
I manually added one grant that was set to zero so I could view it.  It's not necessary so I'll remove it.